provider/resources.go

```diff
@@ -873,14 +873,18 @@ compatibility shim in favor of the new "name" field.`)
 			}
 			r.StateInputs.Properties["name"] = r.InputProperties["name"]
 			postProcessOverlays(spec)
+
+			// Fix a dangling type reference to a resolved resource reference.
+			f := spec.Resources["aws:cloudwatch/logSubscriptionFilter:LogSubscriptionFilter"]
+			f.InputProperties["logGroup"].OneOf[1].Ref = "#/resources/aws:cloudwatch/logGroup:LogGroup"
 		},
 
 		Config: map[string]*tfbridge.SchemaInfo{
 			"assume_role": {
 				MaxItemsOne: tfbridge.True(),
 			},
 			"region": {
-				Type: awsTypeDefaultFile(awsMod, "Region"),
+				Type: "aws:index/Region:Region",
```

Comment:
This now points to the official enum for region. Highlighing here because region is one of the most impactful potentially breaking changes from the lot.